src/util/resourceHelpers.ts

```diff
@@ -41,17 +41,18 @@ export const getContentTypeFromResourceTypes = (resourceTypes: Pick<ResourceType
   return contentTypeMapping.default;
 };
 
-const isLearningPathResourceType = (contentType?: string) => contentType === contentTypes.LEARNING_PATH;
-
-const isConceptType = (contentType?: string) => contentType === "concept";
-const isGlossType = (contentType?: string) => contentType === "gloss";
-const isAudioType = (contentType?: string) => contentType === "audio";
-const isSeriesType = (contentType?: string) => contentType === "series";
+const isLearningPathResourceType = (contentType?: string) =>
+  contentType === "learningpath" || contentType === contentTypes.LEARNING_PATH;
```

Comment:
Er omskrevet til å bruke contentTypes mange steder, men denne røver'n er uten bindestrek og hardkodes. Er den verdien midlertidig og vil forsvinne på sikt, eller må vi håndtere den i overskuelig framtid? Burde den også isåfall vært en "type" som vi kan referere, hvis den trenges flere steder?

Reply:
Funksjonen brukes forskjellige plasser. Eg begynte å rydde opp, men då blei det fort uoversiktlig. Var enklest å godta begge i første omgang. Kan ta det som egen pr i etterkant.